.pre-commit-config.yaml

```diff
@@ -25,10 +25,14 @@ repos:
       - id: poetry-lock
       - id: poetry-install
       - id: pre-commit-install
-      - id: megalinter
-        args: &megalinter-args [--flavor, documentation, --release, v5.16.1]
-      - id: megalinter-all
+      - id: megalinter-incremental
+        args: &megalinter-args [--flavor, documentation, --release, v6.10.0]
+        stages:
+          - commit
```

Comment:
Multiple instances: this would be needed if we used the official MegaLinter hooks, but our version still specifies the stage.